src/ssb_project_cli/ssb_project/app.py

```diff
@@ -131,41 +206,46 @@ def create_project_from_template(projectname: str, description: str):
         quoted,
     ]
     # try:
-    subprocess.run(argv, check=True, cwd=home_dir)
+    subprocess.run(argv, check=True, cwd=home_dir)  # noqa: S603 no untrusted input
     # except subprocess.CalledProcessError:
     #     typer.echo(f"ERROR calling cruft.")
     return project_dir
 
 
 class RepoPrivacy(str, Enum):
+    """Class with predefined privacy enums."""
+
     internal = "internal"
     private = "private"
     public = "public"
 
 
 @app.command()
 def create(
-    project_name: str = typer.Argument(
+    project_name: str = typer.Argument(  # noqa: B008
         ..., help="Prosjekt navn, kun alfanumerisk og underscore"
     ),
-    description: str = typer.Argument(
+    description: str = typer.Argument(  # noqa: B008
         ..., help="En kort beskrivelse av hva prosjektet ditt er for"
     ),
-    repo_privacy: Optional[RepoPrivacy] = typer.Argument(
+    repo_privacy: Optional[RepoPrivacy] = typer.Argument(  # noqa: B008
         RepoPrivacy.internal, help="En kort beskrivelse av hva prosjektet ditt er for"
     ),
-    skip_github: Optional[bool] = typer.Option(
+    skip_github: Optional[bool] = typer.Option(  # noqa: B008
         False, help="Legg denne til hvis man IKKE ønsker å opprette et Github repo"
     ),
-    github_token: Optional[str] = typer.Option(
+    github_token: Optional[str] = typer.Option(  # noqa: B008
         "",
-        help="Ditt Github PAT, følg [link=https://statistics-norway.atlassian.net/wiki/spaces/DAPLA/pages/1917779969/Oppstart+personlig+GitHub-bruker+personlig+kode+og+integrere+Jupyter+med+GitHub#Opprette-personlig-aksesskode-i-GitHub]instruksjonene her[/link] for å skape en",
+        help="""Ditt Github PAT, følg [link=https://statistics-norway.atlassian.net/wiki
```

Comment:
Whitespace is respected inside Docstrings so this will look strange when printed since the indentation will be included as spaces.

I recommend ignoring the flake8 warning or using method 3 here: https://www.golinuxcloud.com/python-multi-line-strings-examples/

Reply:
Disabled flake8 B950, and reverted to old format.